react/features/analytics/middleware.js

```diff
@@ -109,6 +137,26 @@ MiddlewareRegistry.register(store => next => action => {
     case CONFERENCE_JOINED: {
         const { dispatch, getState } = store;
         const state = getState();
+        const config = state['features/base/config'];
+        const { analytics } = config;
+
+        if (analytics.rtcstatsEnabled) {
+            // Once the conference started connect to the rtcstats server and send data.
+            try {
+                RTCStats.connect();
+
+                // The current implementation of rtcstats-server is configured to send data to amplitude, thus
+                // we add identity specific information so we can corelate on the amplitude side. If amplitude is
+                // not configured an empty object will be sent.
+                RTCStats.sendIdentityData({
+                    ...getAmplitudeIdentity(),
+                    ...config
```

Comment:
instead of passing the entire config object, can you explicitly select and pass only the properties that this method needs?

Reply:
The thing is we might want various information from the config, the requirements about what is needed from there can change, so Emil suggested we send the whole config.

Reply:
well.. yeah, but in order to do that you will be pulling updates to RTC stats version into package.json which could as well contain the update to the properties passed through. It's just something we used to do in the past and it was proven to be problematic/unreadable/hard to follow.

Reply:
You're right we can change it when we do an update, however older dumps that we store in s3 prior to those updates won't have the metadata, e.g. say we push our stats to google big query (or another big data store), if we decide to query for something that's not currently in our, lets say config property filter, we wouldn't be able to do that for previously saved stats dumps.
Uploading to a big data store is something we'd want to do in the future. In that scenario querying for data becomes a bit more dynamic, so in my opinion it doesn't hurt to have the data there, considering that it's sent one time and it's not that large.

Reply:
uhm.. I'm missing something here. I though RTC stats relies on few config options to initialize itself like the polling interval. Is there something that it's actually pushing to the data store next to the stats?

Reply:
the fact that we're having this conversation it's only proving my point. Nobody can't tell from the code what is really needed for the RTC stats here. How about having specific section/object defined in config JS for such data? Are you going to just store the entire config.js on the cloud just in case?

Reply:
Sorry for the confusion, I will make the comment more verbose, in order to better explain the reason for the config.
So the config is just sent as metadata to the rtcstats-server, it doesn't do anything to the rtcstats-client or server. The reason we are sending it, is so that we can have queries like: "I want to see all peer-connection that were connected to          shard: 'meet-jit-si-eu-central-1b-s39'", or "All PC that have releaseNumber: xxx". 
Does that make more sense?

Reply:
It makes sense and the comment will do. I still would prefer to include those sections explicitly as we have them defined and they are well known already(`config.deploymentInfo` for example), but I'm not going to block the PR on that.